tests/ui/fn/ok.🔬

```diff
@@ -0,0 +1,10 @@
+[
+    crate Foo {
+        fn no_args<>() -> () where [] { trusted }
+        fn one_args<ty T, lt l>(&l T) -> () where [] { trusted }
+        fn one_return<ty T, lt l, ty U>(&l T) -> U where [] { trusted }
```

Comment:
Leave a comment explaining what Rust signature this signature is equivalent to. this is `fn foo<'a, T, U>(_: &'a T) -> U`, and previously you had different functions.

Also leave a comment explaining what the tests are testing, as I can't tell from the test.